lib/core/src/Cardano/Wallet/Api/Client.hs

```diff
@@ -228,7 +229,7 @@ byronWalletClient =
             , listWallets = _listWallets
             , postWallet = _postWallet
             , putWallet = _putWallet
-            , putWalletPassphrase = error "TODO: putWalletPassphrase"
+            , putWalletPassphrase = error "putWalletPassphrase: not impl for byron"
```

Comment:
This should be `_putByonWalletPassphrase` now, no :) ?

Reply:
I have this problem of using `ByronWalletPutPassphraseData` and not `WalletPutPassphraseData` in `WalletClient` and it contributed to the solution you did not liked  done in commit https://github.com/input-output-hk/cardano-wallet/pull/1495/commits/e485e7bad1fd04c6fab3b7997715a354b0d60e85

Reply:
Okay, I'll look into this :+1: